diagnostic.md

```diff
@@ -48,11 +53,15 @@ Record your responses inside the fenced code blocks below each question.
     value `'123'` inside a Route?
 
     ```md
-    <!-- your response here -->
+    Pass a parameter (params) to the model function that holds the dynamic segment of a route.
+    [params.movie_id - 1]
     ```
```

Comment:
Exactly right to access `params`. `[params.movie_id - 1]` just accesses the element of the array at the correct index, but the magic here is `params.movie_id`